boost.sh

```diff
@@ -913,6 +1022,18 @@ scrunchAllLibsTogetherInOneLibPerPlatform()
                     "$MACOS_BUILD_DIR/$ARCH/libboost_$NAME.a"
             fi
         fi
+
+        if [[ -n $BUILD_MAC_CATALYST ]]; then
+            if [[ "${#MAC_CATALYST_ARCHS[@]}" -gt 1 ]]; then
+                for ARCH in "${MAC_CATALYST_ARCHS[@]}"; do
+                    $MAC_CATALYST_DEV_CMD lipo "mac-catalyst-build/stage/lib/libboost_$NAME.a" \
+                        -thin "$ARCH" -o "$MAC_CATALYST_BUILD_DIR/$ARCH/libboost_$NAME.a"
+                done
+            else
+                cp "mac-catalyst-build/stage/lib/libboost_$NAME.a" \
+                    "$MAC_CATALYST_BUILD_DIR/$ARCH/libboost_$NAME.a"
```

Comment:
`$ARCH` should be `$MAC_CATALYST_ARCHS[0]` here :)

Reply:
You're absolutely right, thank you :)